package exception;

public class ArrayIndexOutOfBoundEx {

	public static void main(String[] args) {
		try
		{
	 int[]a= {1,2,3,5};
	System.out.println(a[2]);

	}
		catch(Exception e)
		{
			
			System.out.println("some Error in code");
		}

}
}
